lib/middlewares/api/authentication.go

```diff
@@ -0,0 +1,49 @@
+package apimiddlewares
+
+import (
+	"go-google-scraper-challenge/errors"
+	helpers "go-google-scraper-challenge/helpers/api"
+	"go-google-scraper-challenge/lib/models"
+	"go-google-scraper-challenge/lib/services/oauth"
+	"strconv"
```

Comment:
Can you please rearrange these ones?

Reply:
Fixed in [255a51d](https://github.com/carryall/go-google-scraper-challenge/pull/127/commits/255a51de22aca1fe82e0108bd17ad93075c881ae)